go/libkb/passphrase_state.go

```diff
@@ -24,20 +22,20 @@ func LoadPassphraseState(mctx MetaContext) (passphraseState keybase1.PassphraseS
 }
 
 // forceRepoll only forces repoll when the state is RANDOM, but not when it is KNOWN.
-func LoadPassphraseStateWithForceRepoll(mctx MetaContext, forceRepoll bool) (passphraseState keybase1.PassphraseState, err error) {
+func LoadPassphraseStateWithForceRepoll(mctx MetaContext, _ bool) (passphraseState keybase1.PassphraseState, err error) {
 	mctx = mctx.WithLogTag("PPSTATE")
-	defer mctx.TraceTimed(fmt.Sprintf("LoadPassphraseState(forceRepoll=%t)", forceRepoll), func() error { return err })()
+	defer mctx.TraceTimed(fmt.Sprintf("LoadPassphraseState()"), func() error { return err })()
+
+	// If we're in standalone mode, we don't get the gregor msg about
+	// passphrase_state changes.  So, force a repoll to the server if the stat
+	// eisn't currently known.
+	forceRepoll := mctx.G().Standalone
```

Comment:
Behavior change: ignore the input and just forcerepoll whenever in standalone mode. Assume gregor has updated us otherwise.